packages/livechat/widget-demo.html

```diff
@@ -48,7 +48,7 @@
 			var h = d.getElementsByTagName(s)[0],
 				j = d.createElement(s);
 			j.async = true;
-			j.src = 'build/rocketchat-livechat.min.js?_=' + Math.random();
+			j.src = 'dist/rocketchat-livechat.min.js?_=' + Math.random();
```

Comment:
Not connected to this task in any way... but needed to update it since with the new monorepo structure, `build` folder is no longer generated & this js file resides in the `dist` folder instead